spring-cloud-context/src/main/java/org/springframework/cloud/context/named/NamedContextFactory.java

```diff
@@ -67,6 +67,12 @@ public void setConfigurations(List<C> configurations) {
 		return new HashSet<>(contexts.keySet());
 	}
 
+	protected void createAndCacheContexts() {
```

Comment:
Sounds good @ryanjbaxter, @spencergibb , let me open an issue here to track downstream spring-cloud-netflix issue, will clean up this PR.